test-cargo-miri/build.rs

```diff
@@ -20,4 +20,9 @@ fn main() {
     println!("cargo:rerun-if-changed=build.rs");
     println!("cargo:rerun-if-env-changed=MIRITESTVAR");
     println!("cargo:rustc-env=MIRITESTVAR=testval");
+
+    // Test that autocfg works.
```

Comment:
Arguably this is testing cargo-miri build script support so having this in test-cargo-miri is the right place IMO.